static/js/map/map.pokemon.js

```diff
@@ -268,6 +274,17 @@ function pokemonLabel(item) {
                 </div>`
         }
 
+        
+        var sizeDisplay = ''
+        if(size){
```

Comment:
On this line, can you put a space between the "if" and the "open parenthesis"?
And also, put a space between the last "close parenthesis" and the "open curly brace"?